app/src/main/java/com/kamron/pogoiv/logic/PokemonNameCorrector.java

```diff
@@ -19,19 +18,19 @@
 public class PokemonNameCorrector {
     private final PokeInfoCalculator pokeInfoCalculator;
     private final HashMap<String, String> userCorrections;
-    /* We don't want memory usage to get out of hand for stuff that can be computed. */
-    private final LruCache<String, PokeDist> cachedCorrections;
 
     public PokemonNameCorrector(PokeInfoCalculator pokeInfoCalculator, Map<String, String> storedUserCorrections) {
         this.pokeInfoCalculator = pokeInfoCalculator;
         userCorrections = new HashMap<>(pokeInfoCalculator.getPokedex().size());
         userCorrections.putAll(storedUserCorrections);
-        userCorrections.put("Sparky", pokeInfoCalculator.get(132).name);
-        userCorrections.put("Rainer", pokeInfoCalculator.get(132).name);
-        userCorrections.put("Pyro", pokeInfoCalculator.get(132).name);
```

Comment:
I actually removed it because it kept correcting my rainer to eevee, even though it's a vaporeon. I should have discussed the change before removing, and I should have kept it in a seperate commit.

My thought was that people will want to know the iv's of an eevee before they decide whether to make it a vaporeon or not, so I think its more likely that a pokemon named "rainer" is an already evolved vaporeon, not an eevee.